src/components/AttentionBox/AttentionBox.tsx

```diff
@@ -72,33 +75,33 @@ const AttentionBox: React.FC<AttentionBoxProps> & {
       )}
       role="alert"
     >
-      {title && (
-        <h2 className={cx(`${ATTENTION_BOX_CSS_BASE_CLASS}__title-container`, `${classNameWithType}__title-container`)}>
-          {!withoutIcon && (
-            <Icon
-              iconType={iconType}
-              ariaHidden
-              clickable={false}
-              icon={icon}
-              className={cx(
-                `${ATTENTION_BOX_CSS_BASE_CLASS}__title-container__icon`,
-                `${classNameWithType}__title-container__icon`
-              )}
-              ignoreFocusStyle
-              iconSize="24"
-              iconLabel={iconLabel}
-            />
-          )}
-          <span
+      {/* @ts-ignore */}
+      <Flex
+        /* @ts-ignore */
+        justify={Flex.justify.START}
+        /* @ts-ignore */
+        align={Flex.align.CENTER}
+        className={cx(`${ATTENTION_BOX_CSS_BASE_CLASS}__title-container`, `${classNameWithType}__title-container`)}
+      >
+        {!withoutIcon && (
+          <Icon
+            iconType={iconType}
+            ariaHidden
+            clickable={false}
+            icon={icon}
             className={cx(
-              `${ATTENTION_BOX_CSS_BASE_CLASS}__title-container__title`,
-              `${classNameWithType}__title-container__title`
+              `${ATTENTION_BOX_CSS_BASE_CLASS}__title-container__icon`,
+              `${classNameWithType}__title-container__icon`
```

Comment:
Spot a little misalignment in storybook, I guess it shouldn't look like exactly this
Before:
<img width="481" alt="image" src="https://user-images.githubusercontent.com/104433616/204127985-27bca268-6cad-4b62-8dd6-8a8e64c42e4f.png">

Now:
<img width="466" alt="image" src="https://user-images.githubusercontent.com/104433616/204127979-f1cb6c99-7248-46d9-a688-128fd0eded4b.png">

Reply:
nice catch!! will fix now

Reply:
@SergeyRoyt done and merging